frontend/lib/src/theme/globalStyles.ts

```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  */
 
-import { css } from "@emotion/react"
+import { css, SerializedStyles } from "@emotion/react"
 import { darken, transparentize, readableColor } from "color2k"
-import { EmotionTheme } from "src/lib/theme"
+import { EmotionTheme } from "src/theme"
 
-export const globalStyles = (theme: EmotionTheme): any => css`
+// This theme should be emotionTheme but needs to be researched more why
+// app doesn't pick up theme properly
+export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
```

Comment:
I don't understand what this comment means. "This theme should be emotionTheme" - should it be named "emotionTheme" or typed as `EmotionTheme` or something else? What does it mean that "app doesn't pick up theme properly"?

More generally, how should someone reading this comment do this research?

Reply:
Ah this is actually resolved. I need to remove this.